packages/trader/src/App/Components/Elements/Modals/UnsupportedContractModal/unsupported-contract-modal.tsx

```diff
@@ -21,7 +21,7 @@ const UnsupportedContractModal = observer(
             <Dialog
                 title={localize('We’re working on it')}
                 confirm_button_text={localize('Stay on {{website_domain}}', { website_domain: website_name })}
-                cancel_button_text={localize('Go to Binary.com')}
+                cancel_button_text={localize('Go to Deriv.com')}
```

Comment:
i think it doesn't make much sense to have this text. 'Stay on Deriv.com' would make more sense. :) Also this button is still redirecting the user to binary `urlFor('user/portfoliows', { legacy: true })`. It should simply close the modal, right? Please check🙏🏻

Reply:
If we are going to change text, then we need this long way with content team. My task was just about replacing Binary ---> Deriv.
Both buttons work as expected: 'Go to Deriv.com' will open Deriv.com and 'Stay on Deriv' will close the modal

Reply:
But will remove `urlFor('user/portfoliows', { legacy: true })`👍🏻

Reply:
shouldn't this be go to Smarttrader?